### PR TITLE
Better Login Interface

### PR DESCRIPTION
**Pull request by Caleb-Littlejohn**
I changed the false signups that were originally there and added user messages when there were issues (trying to create a user with an existing user's email, too weak of a password, etc.). Also implemented a Password change page. I have the barebones of a feedback form in the code commented out to try github issues rather than e-mailing feedback. Additionally, put the table with the user's models in the profile page.

### CODE REVIEW

**Comment by mschrimpf** [comment]
resolves #8

File: benchmarks/view.py [hunk 1]
```diff
@@ -0,0 +1,287 @@
...
+            context = get_context(request.user)
+            return render(request, 'benchmarks/profile.html', context)
+
+    def post(self, request):
+        user = authenticate(request, username=request.POST['username'], password=request.POST['password'])
+        if user is not None:
+            login(request, user)
+
+            # Currently, there are no model submissions with user fields, so for testing
+            # purposes, trying to get the model named "xception" will allow an easy change to the user models.
```
Comment (mschrimpf) [inline benchmarks/view.py]: outdated comment?

File: benchmarks/view.py [hunk 2]
```diff
@@ -0,0 +1,287 @@
...
+        if form.is_valid():
+            # Send an email to the user with the token:
+            mail_subject = request.POST['Subject']
+            message = "{0}\n{1}".format(request.user, request.POST['Feedback'])
+            email = EmailMessage(mail_subject, message, to=['clittlejohn268@gmail.com'])
+            email.send()
+            context = {"activation_email": False, "password_email": False, 'form': LoginForm}
+            return HttpResponseRedirect("../profile")
+        else:
+            return HttpResponseRedirect("../profile")"""
```
Comment (mschrimpf) [inline benchmarks/view.py]: dead code, move to different branch/PR

File: web/settings.py [hunk 3]
```diff
@@ -27,6 +27,14 @@
 
 ALLOWED_HOSTS = []
 
+# Allows E-mail used
+EMAIL_USE_TLS = True
+EMAIL_HOST = 'smtp.gmail.com'
+EMAIL_PORT = 587
+EMAIL_HOST_USER = 'clittlejohn268@gmail.com'
+EMAIL_HOST_PASSWORD = 'newtab15'
```
Comment (mschrimpf) [inline web/settings.py]: don't put your secrets into production ;)

File: benchmarks/forms.py [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+	class Meta:
+		model = UploadPlaceHolder
+		fields = ( 'name', 'zip_file')
+"""
+Feedback Form (email)
+
+class FeedbackForm(forms.Form):
+Subject = forms.CharField(max_length=200, help_text='Required')
+Feedback = forms.CharField(max_length=1000, help_text='Required')
+Feedback.widget = forms.Textarea()"""
```
Comment (mschrimpf) [inline benchmarks/forms.py]: dead code, move to different branch/PR

File: benchmarks/templates/benchmarks/base.html [hunk 5]
```diff
@@ -94,10 +94,10 @@
...
     <div id="navMenu" class="navbar-menu">
         <div class="navbar-end">
-            <a class="navbar-item" href="#leaderboard">Leaderboard</a>
-            <a class="navbar-item" href="#about">About</a>
-            <a class="navbar-item" href="#analysis">Compare</a>
-            <a class="navbar-item" href="#participate">Participate</a>
+            <a class="navbar-item" href="http://www.brain-score.org/#leaderboard">Leaderboard</a>
+            <a class="navbar-item" href="http://www.brain-score.org/#about">About</a>
+            <a class="navbar-item" href="http://www.brain-score.org/#analysis">Compare</a>
+            <a class="navbar-item" href="http://www.brain-score.org/#participate">Participate</a>
```
Comment (mschrimpf) [inline benchmarks/templates/benchmarks/base.html]: IIRC in current `master`, these should resolve to the currently used URL which lets this run for development as well as in production

File: benchmarks/view.py (hunk 1 above)
Comment (Caleb-Littlejohn) [reply benchmarks/view.py]: Yes

File: benchmarks/templates/benchmarks/base.html (hunk 5 above)
Comment (Caleb-Littlejohn) [reply benchmarks/templates/benchmarks/base.html]: Since there are multiple different pages using this navbar, I have it redirecting to the main page.

**Comment by mschrimpf** [comment]
looks good, but unit tests fail with `sqlite3.OperationalError: no such table: auth_group`

**Pull request merged**